projects/experimental/directives/card/card.styles.less

```diff
@@ -31,4 +32,83 @@
     [tuiSubtitle] {
         max-width: 100%;
     }
+
+    &[data-size='l'] {
+        --t-size: 100%;
+        height: auto;
+        background: var(--tui-base-01);
+        border-radius: var(--t-radius);
+        padding: var(--t-padding);
+        gap: var(--t-gap);
+        box-shadow: 0 0.375rem 2.125rem 0 rgba(0, 0, 0, 0.12);
+        font-size: 1rem;
+
+        & > * {
+            width: 100%;
+        }
+
+        [tuiCell] {
+            --t-padding: 0.5rem 0;
+            box-sizing: border-box;
+        }
+
+        &[data-appearance='normal'] {
+            --t-gap: 1.25rem;
+            --t-radius: 1.5rem;
+            --t-padding: 1rem 1.5rem;
+
+            & > *:first-child:not(:last-child) {
```

Comment:
Could you clarify these 2 rules with vertical margins? When are they used?

Reply:
<img width="1132" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/9dab6889-0104-48c1-89a8-aa2450a2c6cb">

Reply:
fixed